### PR TITLE
New package: plex-media-server-1.15.3.876

### LINKED ISSUE #10601: Package Request: Plex Media Server
URL: https://www.plex.tv/
Description: `Plex Media Server`
Release: https://www.plex.tv/media-server-downloads/#plex-media-server

The client is already packaged: https://github.com/void-linux/void-packages/blob/e866a7ee299f80a652219e510a16fd561eb356e4/srcpkgs/plex-media-player/template
Arch PKGBUILD: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=plex-media-server

### PR DESCRIPTION
**Pull request by Anachron**
Fixes #10601

### CODE REVIEW

**Comment by Anachron** [comment]
All builds except [i686](https://travis-ci.org/void-linux/void-packages/jobs/516574265#L1030) are successful. I have no idea what's up with that one.

File: srcpkgs/plex-media-server/template [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+nopie="yes"
+nopie_files="/usr/lib/plexmediaserver/Plex DLNA Server"
+
+do_extract() {
+	fn=$(basename "${distfiles}")
+	ar x "${XBPS_SRCDISTDIR}/${pkgname}-${version}/${fn}"
+	tar xf "data.tar.xz"
+	mv usr/{s,}bin
+	sed -i 's/export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR.*/export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR="${HOME}\/.pms"/g' usr/bin/start_pms
```
Comment (maxice8) [inline srcpkgs/plex-media-server/template]: this line can be cut with \

to be more readable.

File: srcpkgs/plex-media-server/template [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			distfiles="https://downloads.plex.tv/${pkgname}-new/${version}-${_suffix}/debian/plexmediaserver_${version}-${_suffix}_arm64.deb"
+			checksum="3c343bb7f6f335637a1fec15d790e53a9a8e73ec66918ae11f41b4d11073d110"
+			;;
+esac
+
+nopie="yes"
+nopie_files="/usr/lib/plexmediaserver/Plex DLNA Server"
+
+do_extract() {
+	fn=$(basename "${distfiles}")
```
Comment (maxice8) [inline srcpkgs/plex-media-server/template]: can be ${distfiles##*/}

File: srcpkgs/plex-media-server/template [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	ar x "${XBPS_SRCDISTDIR}/${pkgname}-${version}/${fn}"
+	tar xf "data.tar.xz"
```
Comment (maxice8) [inline srcpkgs/plex-media-server/template]: doesn't need to be quoted

**Comment by maxice8** [comment]
license is Proprietary and don't need to vendor it

**Comment by Anachron** [comment]
Should be resolved now. I don't know what you mean with "license is Proprietary and don't need to vendor it". Many packages that are proprietary ship their license in Void Linux. If you want me to remove it I'll do that.

File: srcpkgs/plex-media-server/template [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Template file for 'plex-media-server'
+pkgname=plex-media-server
+version=1.15.3.876
+revision=1
+_suffix=ad6e39743
+archs="i686 x86_64 armv7 aarch64"
+create_wrksrc=yes
+short_desc="Plex Media Server"
+maintainer="Anachron <gith@cron.world>"
+license="custom"
```
Comment (maxice8) [inline srcpkgs/plex-media-server/template]: Put `Proprietary` as string

File: srcpkgs/plex-media-server/template [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+	mv usr/{s,}bin
+	sed -i 's/export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR.*/'\
+'export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR="${HOME}\/.pms"/g' usr/bin/start_pms
+	rm -r etc/{apt,init}
+}
+
+do_install() {
+	vcopy etc /
+	vcopy usr /
+	vlicense "${FILESDIR}/LICENSE" LICENSE-plex-media-server
```
Comment (maxice8) [inline srcpkgs/plex-media-server/template]: no need to rename

**Comment by Anachron** [comment]
All fixed now.

**Comment by belialzxz** [comment]
Trying to install this on my machine (x64) and I'm getting the following error:

/void-packages/srcpkgs/plex-media-server/template: line 44: tar: command not found
=> ERROR: plex-media-server-1.15.3.876_2: do_extract: 'tar xf data.tar.xz' exited with 127
=> ERROR:   in do_extract() at srcpkgs/plex-media-server/template:44

the tar command works when I try it in the CLI, so it is there, and it shows as installed when looking it up via xbps-query. export command works as well from CLI when I tested it.

**Comment by pullmoll** [comment]
@belialzxz Should be fixed with plex-media-server-1.15.3.876_3 now.

**Comment by belialzxz** [comment]
> @belialzxz Should be fixed with plex-media-server-1.15.3.876_3 now.

Awesome!!! Thanks! It built with not issue.

**Pull request merged**